.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: c
+sudo: required
```

Comment:
From the Travis CI docs, this seems to get the boot time from 1-6s to 20-52s.

https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

In your experience on other projects, does that really make the tests that long to run?